docs/docs/winning-over-content-creators.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Winning Over Content Creators"
+issue: https://github.com/gatsbyjs/gatsby/issues/18281
+---
+
+Content creators are probably the heaviest end-users of your Gatsby projects because they create and manage the content used in your project. If you're a developer, you may find that simply talking to content creators about why _you_ like Gatsby is enough to get them interested. Below are topics a content creator cares about and how Gatsby can improve these.
+
+Some specific things that content creators care about include:
+
+- **Using tools they are familiar with.** The content creator probably has tools and workflows that they are already using, like a CMS. Gatsby could be used to integrate and possibly improve that workflow.
+- **Consistency** Consistency is key for a content creator, so their content matches the related content and branding.
+- **Audience**. Probably the biggest thing the content creator cares about is the audience, for who are they creating content?
+- **Flexible content**. Content creators care about personalizing content based on the audience that is viewing it, and creating multiple kinds of content.
+
+## Benefits
+
+Content creators may be curious about topics such as current workflow, new possibilities, flexibility.
+
+### Current workflow
+
+> Gatsby integrates with a bunch of data sources and CMS's, there are ways to integrate your current workflow using Gatsby. If you love using your current CMS, great! Gatsby doesn’t decide your workflow, you do.
+
+### Possibilities
+
+> Multiple data sources make it possible to create your content wherever you want to. You could add YouTube videos or any other external API as content to Gatsby. Reach your audience in new ways.
```

Comment:
I suggest revising to something like, "With Gatsby, you can pick whichever tools work best for your site and your content - choose the best CMS, the best ecommerce tool, the best video hosting, and build your own content stack without worrying about unreliable plugins or the politics of software partnerships." Some of my suggestion might be a little too snarky...